app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Can%20reset%20password.-,Modify%20the%20views%20in%20devise%20for%20registration%20and%20login%20to%20match%20the%20wireframes%20from%20the%20Sneak%20Peek%20and%20your%20styling.,-Need%20a%20big), Please edit the signup page design to match the given wireframe design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/rmiyoyo/blog-app/assets/68796232/259cbd76-7051-4b17-9ab2-60c1e227b2fb)  | ![image](https://github.com/rmiyoyo/blog-app/assets/68796232/f94ace99-12d2-4a75-bef6-cc2515dad4fd)  |